src/config/mozambique/layers.json

```diff
@@ -4,8 +4,12 @@
     "path": "data/mozambique/moz_bnd_adm1_WFP.json",
     "opacity": 0.8,
     "admin_code": "source_id",
-    "admin_level_names": ["adm1_name"],
-    "admin_level_local_names": ["adm1_name"],
+    "admin_level_names": [
```

Comment:
not a huge deal but it looks like we don't have the same linter rules, which is created a bit of xtra changes

Reply:
@ericboucher - Thanks for the fix. The config files to get unwieldy and challenging to navigate quickly with multiple parameters split over new lines. Thoughts on revisiting the linter rules to make this a bit easier to navigate?